doc/rst/language/spec/conversions.rst

```diff
@@ -218,54 +175,235 @@ type ``C`` is allowed when ``D`` is a subclass of ``C``.
 
 Any combination of these three conversions is allowed.
 
+.. _Subtype_Arg_Conversions:
 .. _Implicit_Type_Arg_Conversions:
+.. _Implicit_Generic_Type_Conversions:
 
-Implicit Type Argument Conversions
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Implicit Subtype Conversions
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-An implicit type argument conversion applies only when a type actual is
-passed to a formal with the ``type`` intent. This includes the ``this``
-formal of a type method. In this case, a subset of Implicit Class
-Conversions (:ref:`Implicit_Class_Conversions`) applies, in
-addition to Implicit Conversions To Generic Types
-(:ref:`Implicit_Generic_Type_Conversions`).
+An implicit subtype conversion applies when the type being converted to
+is a subtype of the type of the value. In particular, it can apply when
+the type of an actual argument is a subtype of the type of the formal
+argument. In that case, we might say that type ``T1`` is the type
+of the actual argument and type ``T2`` is the type of the formal.
 
-   .. note::
-      
-      *Future:*
+Given any two types ``T1`` and ``T2``, the type ``T1`` is considered to be a
+subtype of a type ``T2`` if:
 
-      The details are forthcoming.
+ * ``T2`` is a generic type (:ref:`Generic_Types`) and
+   the ``T1`` is an instantiation that type
+ * ``T1`` is a class type that inherits from the type ``T2``
+ * or a combination of the two.
 
-.. _Implicit_Statement_Bool_Conversions:
+When a type actual is passed to a formal with ``type`` intent and a
+declared type, an implicit subtype conversion occurs from the actual type
+to the formal type argument if and only if the actual type is a subtype
+of the declared formal type.
 
```

Comment:
I'm thinking I'll just change this "Additionally, when an actual is passed to a formal with generic type" section to use source type/target type language instead of talking about actuals and formals.

Reply:
I changed the entire section to 

```

Implicit Subtype Conversions
~~~~~~~~~~~~~~~~~~~~~~~~~~~~

An implicit subtype conversion is allowed when the source type is a
subtype of the target type.

Given any two types ``T1`` and ``T2``, the type ``T1`` is considered to be a
subtype of a type ``T2`` if:

 * ``T2`` is a generic type (:ref:`Generic_Types`) and
   the ``T1`` is an instantiation that type
 * ``T1`` is a class type that inherits from the the class ``T2``
 * or a combination of the two.

The below examples with the ``type`` intent demonstrate subtype 
conversions.

... examples ...
```

which seems better factored to me. I have an update to the resolution details that talks about type formals.